subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AndroidSantaTrackerSmokeTest.groovy

```diff
@@ -94,20 +95,11 @@ class AndroidSantaTrackerSmokeTest extends AbstractSmokeTest {
 
     private void setupCopyOfSantaTracker(TestFile targetDir) {
         copyRemoteProject("santaTracker", targetDir)
-
-        def settingsFile = targetDir.file("settings.gradle")
-        settingsFile.text = """
-            gradleEnterprise {
-                buildScan {
-                    server = "https://e.grdev.net/"
-                    captureTaskInputFiles = true
-                }
-            }
-        """ + settingsFile.text
+        EnterprisePluginFixture.publishScansToInternalServer(targetDir.file("settings.gradle"))
     }
 
     private BuildResult buildLocation(File projectDir) {
-        runner("assembleDebug", "--scan")
+        runner("assembleDebug")
```

Comment:
Why do we want to publish the scan for the smoke test?

Reply:
Hm. We probably don't want to only publish from this one test, so I should remove it from here. (I guess `--scan` was originally added for debugging.) 

But we might want to do this for all smoke tests, because why not? But that should be a separate PR.